build issue in the master

there is build issue in the master branch, 

../src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c: In function ‘MPIDI_XPMEM_init_world’:
../src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c:57:64: error: macro "MPIR_CHKPMEM_MALLOC" passed 6 arguments, but takes just 3
   57 |                         mpi_errno, "xpmem segmaps", MPL_MEM_SHM);
      |                                                                ^
In file included from ../src/include/mpiimpl.h:166,
                 from ../src/mpid/ch4/include/mpidimpl.h:17,
                 from ../src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c:6:
../src/include/mpir_mem.h:143: note: macro "MPIR_CHKPMEM_MALLOC" defined here
  143 | #define MPIR_CHKPMEM_MALLOC(pointer_,nbytes_,class_) \
      | 
../src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c:55:5: error: ‘MPIR_CHKPMEM_MALLOC’ undeclared (first use in this function)
   55 |     MPIR_CHKPMEM_MALLOC(MPIDI_XPMEMI_global.segmaps, MPIDI_XPMEMI_segmap_t *,
      |     ^~~~~~~~~~~~~~~~~~~

it looks that the macrodefine changed, but the corresponding code in the xpmem doesn't

--- a/src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c
+++ b/src/mpid/ch4/shm/ipc/xpmem/xpmem_init.c
@@ -52,9 +52,8 @@ int MPIDI_XPMEM_init_world(void)
 
     /* Initialize segmap for every local processes */
     MPIDI_XPMEMI_global.segmaps = NULL;
-    MPIR_CHKPMEM_MALLOC(MPIDI_XPMEMI_global.segmaps, MPIDI_XPMEMI_segmap_t *,
-                        sizeof(MPIDI_XPMEMI_segmap_t) * MPIR_Process.local_size,
-                        mpi_errno, "xpmem segmaps", MPL_MEM_SHM);
+    MPIR_CHKPMEM_MALLOC(MPIDI_XPMEMI_global.segmaps,
+                        sizeof(MPIDI_XPMEMI_segmap_t) * MPIR_Process.local_size, MPL_MEM_SHM);
     for (i = 0; i < MPIR_Process.local_size; i++) {
         MPIDU_Init_shm_get(i, sizeof(xpmem_segid_t), &MPIDI_XPMEMI_global.segmaps[i].remote_segid);
         MPIR_ERR_CHKANDJUMP(MPIDI_XPMEMI_global.segmaps[i].remote_segid == -1, mpi_errno,

